proto/viam/app/build/v1/build.proto

```diff
@@ -83,3 +99,74 @@ message ListJobsResponse {
   // jobs is ordered by (build start time, alphabetical platform).
   repeated JobInfo jobs = 1;
 }
+
+message RepoLink {
+  string app_link_id = 1;
+  // OrgID of the module
+  string org_id = 2;
+  // public namespace of the module
+  optional string namespace = 3;
+  // ID of the module (just the name, not the dotted org:module form)
+  string module_id = 4;
+  // git repo in owner/repository form
+  string repo = 5;
+  // email of the viam user who created this
+  string viam_user = 6;
```

Comment:
Will it be an issue if the viam user is deleted?

Reply:
this is just for tracking history; the authorization is done through the OAuth app link, which is permissioned per viam org